packages/@react-spectrum/s2/src/NotificationBadge.tsx

```diff
@@ -39,10 +39,17 @@ export interface NotificationBadgeProps extends DOMProps, AriaLabelingProps, Sty
   value?: number | null
 }
 
-export const NotificationBadgeContext = createContext<ContextValue<Partial<NotificationBadgeProps>, DOMRefValue<HTMLDivElement>>>(null);
+interface NotificationBadgeContextProps extends Partial<NotificationBadgeProps> {
+  isDisabled?: boolean
+}
+
+export const NotificationBadgeContext = createContext<ContextValue<Partial<NotificationBadgeContextProps>, DOMRefValue<HTMLDivElement>>>(null);
 
 const badge = style({
-  display: 'flex',
+  display: {
+    default: 'flex',
+    isDisabled: 'none'
```

Comment:
Do we need `isDisabled` as a prop on a standalone notification badge, or could we just set `display: none` in ActionButton in the `styles`?

Reply:
well...`display` isn't part of StyleProps (or I don't think it is?). we could also opt for UNSAFE_style

Reply:
missed that this was only on the context.